Content/ReleaseBriefings/4_6/Section_GeneralEnhancements.tex

```diff
@@ -85,3 +85,80 @@
+    \item \texttt{basic\_simd\& operator>>=(basic\_simd\&, U\&\&)}
+    \item \texttt{basic\_simd\& operator<<=(basic\_simd\&, U\&\&)}
+  \end{itemize}
+
+  \vspace{5pt}
+  
+  \begin{itemize}
+    \item \texttt{T reduce\_min(const basic\_simd\& x)}
+    \item \texttt{T reduce\_max(const basic\_simd\& x)}
+    \item \texttt{T reduce(const basic\_simd\& x, const mask\_type\& mask, T identity\_element, BinaryOperation binary\_op)}
+      \begin{itemize}
+        \item Supported binary operations are: std::plus, std::multiplies, std::bit\_and, std::bit\_or and std::bit\_xor
+        \item std::plus is used if binary op is not specified
+      \end{itemize}
+  \end{itemize}
+
+\end{frame}
+%==========================================================================
+\begin{frame}[fragile]{Avoid storing a view in RandomAccessIterator}
+
+  In Kokkos 4.5, \texttt{Kokkos::sort} was changed to use raw pointers on host
+  \begin{itemize}
+    \item Resolves performance issue of \texttt{Kokkos::sort} on host when using \texttt{RandomAccessIterator}
+    \item Caused by reference counting and view access in \texttt{RandomAccessIterator}
+  \end{itemize}
+
+  \vspace{10pt}
+  
+  Following revisions are made in Kokkos 4.6
+  \begin{itemize} 
+    \item Reverted previous changes in \texttt{Kokkos::sort}
+    \item Replaced views with raw pointers in \texttt{RandomAccessIterator} instead
+    \begin{itemize}
+      \item Prevents \texttt{RandomAccessIterator} from participating in view reference counting
+    \end{itemize}
+  \end{itemize}
+
+\end{frame}
+%==========================================================================
+\begin{frame}[fragile]{Print support for system allocated memory}
+  \texttt{print\_configuration} outputs if system allocated memory is accessible on GPU
+
+  \vspace{5pt}
+
+  \begin{code}
+    Device Execution Space:
+      KOKKOS_ENABLE_HIP: yes
+    HIP Options:
+      KOKKOS_ENABLE_HIP_RELOCATABLE_DEVICE_CODE: no
+    
+    Runtime Configuration:
+      ...
+
+    Kokkos::HIP[ 0 ] gcnArch gfx942:sramecc+:xnack+, Total Global Memory: 94.08 GiB,
+    Shared Memory per Block: 64 KiB,
+    APU or dGPU: APU, Is Large Bar: 1, Supports Managed Memory: 1,
+    XNACK environment variable set: yes
+    Kernel reports HMM module via `CONFIG_HMM_MIRROR=y` in `/boot/config`: yes
+    Architecture capable of accessing system allocated memory: 1,
+    System allows accessing system allocated memory on GPU: 1,
+    Wavefront Size: 64 : Selected
+  \end{code}
+    
+    % macro  KOKKOS_ENABLE_HIP : defined
+    % macro  HIP_VERSION = 60342134 = version 6.3.42134
+    % macro KOKKOS_ENABLE_ROCTHRUST : defined
+    % macro KOKKOS_ENABLE_IMPL_HIP_MALLOC_ASYNC: yes
+    % Kokkos::HIP[ 0 ] gcnArch gfx942:sramecc+:xnack+, Total Global Memory: 94.08 GiB,
+    % Shared Memory per Block: 64 KiB,
+    % APU or dGPU: APU, Is Large Bar: 1, Supports Managed Memory: 1,
+\end{frame}
+%==========================================================================
```

Comment:
Hmm not sure if I would give this so much room. The problem is, that it might raise more questions than we like. As CUDA and HIP allow directly accessing system allocated memory on the gpu users might be curious and start asking questions. 
Maybe it is best just mention this as a small sidenote ... especially since the detection we do for HIP might not work in containers and if it does it is no guarantee

Reply:
Do you suggest any changes on the slide? I agree that we shouldn't go into too much detail into this when presenting. It's probably sufficient to simply mention that a line has been added to the output of `print_configuration()` and move on.

Reply:
I did not suggest changes. Yeah, I would be fine with saying that we try to detect if the gpu can access system allocated memory and include this info in our print